src/app/layouts/common-layout/common-layout.routing.ts

```diff
@@ -94,6 +95,6 @@ export const CommonLayoutRoutes: Routes = [
     component: DepositsComponent,
     canActivate: [AuthGuard],
     data: { title: 'My Deposits', tags: [] }
-  }
-
+  },
+  { path: 'clients', component: ClientsComponent, canActivate: [AuthGuard] }
```

Comment:
Since Chryce has set up the framework for adding page metadata, You can take advantage of that so we do not have to add a separate task after your task gets merged. You may want to sync with Chryce on that.

Reply:
@Victoradukwu I have had sync with Chryce and I have implemented your suggested modifications